knowledge-base/how-to-change-report-server-iis-user.md

```diff
@@ -0,0 +1,70 @@
+---
+title: How to Create a limited IIS User for Report Server and Service Agent
+description: this tutorial shows how to use a lower privlidged user to host Report Server in IIS
+type: how-to
+page_title: How to change IIS user for Report Server and Service Agent
+slug: how-to-change-report-server-iis-user
+tags: report server, iis, user permissions, iis permissions
+res_type: kb
+---
+
+## Environment
+
+|  |  |
+|---------|-----------------|
+| Product | Progress® Telerik® Report Server |
+| Version | 10.1.24.514 |
+
+## Description
+
+This tutorial will show you how to change the Report Server's IIS application pool and service agent to use an identity with limited permissions.
+
+## Solution
+
+1. Create a new **Windows** User with limited permissions.
+
+    ![](images/change-iis-user/1-add-new-user.png)
+
+    *For more help, see [Microsoft Docs - Manage User Accounts in Windows](https://support.microsoft.com/en-us/windows/manage-user-accounts-in-windows-104dc19f-6430-4b49-6a2b-e4dbd1dcdf32).*
+
+2. Go to Telerik Report Server's installation directory (*C:\Program Files (x86)\Progress\Telerik Report Server*) and give `RSUser` full access permissions to the **Telerik.ReportServer.Web** and **Services** subfolders.
+
+      ![](images/change-iis-user/2-add-permissions-to-rsuser.png)
+
+      *Ensure the user does not have read or write permissions to any other folders. For more help, see [Microsoft Q&A - How do I set up user account and manage permissions](https://learn.microsoft.com/en-us/answers/questions/1389054/how-do-i-set-up-user-accounts-and-manage-permissio).*
+
+3. Open Internet Information Services (IIS) Manager (`Windows Key` + `R` to open the Run window, then enter `inetmgr`).
```

Comment:
I recently realized that's the boomers' way to start IIS. Young people just type 'IIS' in the search bar :)

Reply:
LOL, I do use the start menu myself.  "Tap start key => type in IIS =>  click Enter". It's faster than Run for me. I figure by putting the instructions in parenthesis, it is clear that we're sharing one way to do it, but the instruction is to just "Open IIS Manager"